src/ui/auth/ldap/ldap.go

```diff
@@ -26,7 +27,9 @@ import (
 )
 
 // Auth implements AuthenticateHelper interface to authenticate against LDAP
-type Auth struct{}
+type Auth struct {
+	auth.DefaultAuthenticateHelper
```

Comment:
I understand the intention but a little concerned about this approach, let's discuss.